.github/workflows/compute.yml

```diff
@@ -41,56 +37,6 @@ jobs:
         shell: cat {0}
 
       - name: Execute directive
-        run: npx tsx ./src/main.ts
-        id: automated-merging
+        uses: ./
```

Comment:
actually it's dangerous to display everything because that includes `authToken`, so it'd be a good idea to remove it from the payload before displaying it

Reply:
This is not displayed, it gets transformed to `***`
https://github.com/ubiquibot/automated-merging/actions/runs/10468768824/job/28990372701#step:5:13

Reply:
oh that's nice, I guess Github detects strings that contain `token`